staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/controller.go

```diff
@@ -72,6 +78,10 @@ func NewAggregationController(downloader *Downloader, openAPIAggregationManager
 	}
 
 	c.syncHandler = c.sync
+	// Add delegation target API services to queue, to aggregate dynamic OpenAPI spec from delegation target
+	for _, name := range openAPIAggregationManager.GetDelegationAPIServiceNames() {
+		c.queue.AddAfter(name, time.Second)
```

Comment:
why after a second? Why not immediately?

Reply:
Was following the [existing code](https://github.com/kubernetes/kubernetes/blob/f48e18faa4dc035cc927c6a2b34c83c8475b55fa/staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/controller.go#L156) for aggregating an APIService. 

I don't see strong benefit of doing one way verses the other when we add/update an APIService. The delaying interface makes more sense when we deal with [sync reaction](https://github.com/kubernetes/kubernetes/blob/f48e18faa4dc035cc927c6a2b34c83c8475b55fa/staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/controller.go#L114-L123) to avoid hot-loop. 

Maybe we chose to wait a second because we have just aggregated the spec and don't want to pull the HTTP handler again immediately (we will likely get the same ETag).

Reply:
Just want to make sure this second is not a magic number, i.e. it breaks with zero or if the bootstrapping takes longer than the given time.